custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -801,8 +843,10 @@ def _maybe_build_image(self, docker_image_or_directory):
                 for line in client_docker_low_level.build(
                     path=docker_image_or_directory, rm=True, tag=tag
                 ):
-                    json_lines.append(json.loads(line.decode("utf-8").strip()))
```

Comment:
Every line is output from docker build. In Linux I only saw every line to be json string. 
But on Mac @timsetsfire , encountered that line can be a string with to jsons separated with `\n`. So handle this case.